lib/livebook_web/live/app_helpers.ex

```diff
@@ -102,13 +102,35 @@ defmodule LivebookWeb.AppHelpers do
         ]}
       />
       <.radio_field label="Base image" field={@form[:docker_tag]} options={docker_tag_options()} />
```

Comment:
Can we make it a select? I am sure this will  have many more options over time.

Reply:
Sounds good

Reply:
Ah, you meant the Docker tag. Should clustering also be a select, or no?

Reply:
Sorry, I commented out in the wrong place. Base image should be radio. Clustering should be select. Base image will always be 2 or 3 options max. Clustering will grow :)